okta/services/idaas/resource_okta_network_zone.go

```diff
@@ -396,3 +436,66 @@ func mapNetworkZoneToState(d *schema.ResourceData, data *v5okta.ListNetworkZones
 	}
 	return err
 }
+
+// updateNetworkZoneWithExemptList makes a custom HTTP request to update a network zone
+// with the useAsExemptList field included in the JSON body
+func updateNetworkZoneWithExemptList(ctx context.Context, meta interface{}, zoneID string, payload v5okta.ListNetworkZones200ResponseInner) error {
+	// Get the configuration from meta
+	cfg := meta.(*config.Config)
+
+	// Build the URL (no query parameter needed)
+	// The cfg.Domain is just the base domain like "okta.com",
+	// but we need the full org URL like "https://trial-7001215.okta.com"
+	baseURL := fmt.Sprintf("https://%s.%s", cfg.OrgName, strings.TrimSuffix(cfg.Domain, "/"))
+	endpoint := fmt.Sprintf("/api/v1/zones/%s", zoneID)
+	fullURL := baseURL + endpoint
+
+	// Convert SDK payload to map for manipulation
+	jsonData, err := json.Marshal(payload)
+	if err != nil {
+		return fmt.Errorf("failed to marshal payload: %v", err)
+	}
+
+	// Parse JSON into a map so we can add the useAsExemptList field
+	var payloadMap map[string]interface{}
+	if err := json.Unmarshal(jsonData, &payloadMap); err != nil {
+		return fmt.Errorf("failed to unmarshal payload: %v", err)
+	}
+
+	// Add the useAsExemptList field to the JSON payload
+	payloadMap["useAsExemptList"] = true
+
+	// Re-marshal with the added field
+	finalJsonData, err := json.Marshal(payloadMap)
+	if err != nil {
+		return fmt.Errorf("failed to marshal final payload: %v", err)
+	}
+
+	// Create the HTTP request
+	req, err := http.NewRequestWithContext(ctx, "PUT", fullURL, bytes.NewBuffer(finalJsonData))
```

Comment:
Because we're using `http` here directly, I don't think we'll have any of the rate limit awareness that would be otherwise supported via the SDK?

This might be acceptable, I don't think that's my call to make

We might also be able to construct a custom API call using the SDK if it exposes the necessary tooling